docs/source/install/deb.rst

```diff
@@ -31,10 +31,10 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
     sudo apt-get install -y gnupg-curl
     sudo apt-get install -y curl
 
-    # Add key and repo for the latest stable MongoDB (3.2)
-    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EA312927
-    sudo sh -c "cat <<EOT > /etc/apt/sources.list.d/mongodb-org-3.2.list
-    deb http://repo.mongodb.org/apt/ubuntu $(lsb_release -c | awk '{print $2}')/mongodb-org/3.2 multiverse
+    # Add key and repo for the latest stable MongoDB (3.4)
+    wget -qO - https://www.mongodb.org/static/pgp/server-3.4.asc | sudo apt-key adv -
```

Comment:
`sudo apt-key add`,
to sync up with https://github.com/StackStorm/st2-packages/pull/479 fix